src/i18n/z_trans.erl

```diff
@@ -254,6 +254,8 @@ default_language(Context) ->
 
 %% @doc check if the two/three letter code is a valid language
 -spec is_language(Language :: string() | binary()) -> boolean().
+is_language("ppo") -> true; %papiamento
+is_language("ppu") -> true; %papiamentu
```

Comment:
I think that for this to work correctly we also need to add the same clauses for binary strings:
```suggestion
is_language("ppo") -> true; %papiamento
is_language(<<"ppo">>) -> true; %papiamento
is_language("ppu") -> true; %papiamentu
is_language(<<"ppu">>) -> true; %papiamentu
```